Cargo.toml

```
@@ -1,2 +1,2 @@
 [workspace]
-members = ["crates/*", "xtask"]
+members = ["crates/*", "xtask", "examples/*"]
```

Can we sort alphabetically?

```suggestion
members = ["crates/*", "examples/*", "xtask"]
```

---

Cargo sort should do this, is this just failing because it's panicking before this maybe?

---

Maybe that is related to the crash even!